jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -68,6 +80,15 @@
 	 */
 	public static final String DIAGNOSTICS_GC_CLASS_HISTOGRAM = "GC.class_histogram"; //$NON-NLS-1$
 
+	/**
+	 * Commands to generate dumps of various types
+	 */
+	public static final String DIAGNOSTICS_DUMP_HEAP = "Dump.heap"; //$NON-NLS-1$
+	public static final String DIAGNOSTICS_GC_HEAP_DUMP = "GC.heap_dump"; //$NON-NLS-1$
+	public static final String DIAGNOSTICS_DUMP_JAVA = "Dump.java"; //$NON-NLS-1$
+	public static final String DIAGNOSTICS_DUMP_SNAP = "Dump.snap"; //$NON-NLS-1$
+	public static final String DIAGNOSTICS_DUMP_SYSTEM = "Dump.system"; //$NON-NLS-1$
```

Comment:
Is there a plan to refer these declaration outside? otherwise they can be (package) private.